### ISSUE TITLE
Openpose tensorrt implementation

### ISSUE THREAD

**Issue by HoangTienDuc** [zerollzeng/tensorrt-zoo#41]
Hello @zerollzeng 
Your work is awesome.
Do you have any python implementation for openpose

**Comment by zerollzeng** [comment]
No, openpose in tensorrt-zoo is ported from the official openpose repo. which is caffe based.
